pkg/controller/operandconfig/operandconfig_controller.go

```diff
@@ -20,6 +20,8 @@ import (
 	"context"
 	"time"
 
+	operatorv1alpha1 "github.com/IBM/operand-deployment-lifecycle-manager/pkg/apis/operator/v1alpha1"
+	fetch "github.com/IBM/operand-deployment-lifecycle-manager/pkg/controller/common"
```

Comment:
@DanielXLee Should we divide the internal import from the third party library?

Reply:
okay, divide them is better.